### PR TITLE
[Contributions Dashboard] Update contribute tab for donor history

### PR DESCRIPTION
**Pull request by cooltey**
**Phabricator:**
https://phabricator.wikimedia.org/T376435

### CODE REVIEW

**Comment by cooltey** [comment]
Hi @dbrant and @Williamrai, this is ready for a quick review and merge to the feature branch.

File: app/src/main/java/org/wikipedia/main/MainActivity.kt [hunk 1]
```diff
@@ -71,21 +78,51 @@ class MainActivity : SingleFragmentActivity<MainFragment>(), MainFragment.Callba
...
+            if (tab == NavTab.EDITS) {
+                ImageRecommendationsEvent.logImpression("suggested_edit_dialog")
+                PatrollerExperienceEvent.logImpression("suggested_edits_dialog")
+                if (ContributionsDashboardHelper.contributionsDashboardEnabled) {
+                    titleText = if (AccountUtil.isLoggedIn) {
+                        AccountUtil.userName
+                    } else {
+                        getString(R.string.contributions_dashboard_logged_out_user)
+                    }
+                    binding.donorBadge.setup(object : DonorBadgeView.Callback {
```
[the remark this answers is not in the document]
Comment (cooltey) [reply app/src/main/java/org/wikipedia/main/MainActivity.kt:102]: I think there's no specific action for the `donor` status.

File: app/src/main/res/layout/fragment_suggested_edits_tasks.xml [hunk 2]
```diff
@@ -54,217 +43,256 @@
...
+                                    android:id="@+id/learnMoreButton"
+                                    android:layout_width="wrap_content"
+                                    android:layout_height="wrap_content"
+                                    style="@style/App.Button.Secondary"
+                                    android:text="@string/suggested_edits_learn_more"
+                                    android:layout_marginTop="6dp"
+                                    app:layout_constraintTop_toBottomOf="@id/taskDescription"
+                                    app:layout_constraintStart_toStartOf="@id/taskDescription"
+                                    app:layout_constraintBottom_toBottomOf="parent"
+                                    app:icon="@drawable/ic_open_in_new_black_24px"/>
```
Comment (Williamrai) [inline app/src/main/res/layout/fragment_suggested_edits_tasks.xml:265]: this can be removed.

Comment (cooltey) [reply app/src/main/res/layout/fragment_suggested_edits_tasks.xml:262]: Done!

**Comment by dbrant** [comment]
@cooltey Just a couple of conflicts, otherwise lgtm

**Review by Williamrai — APPROVED** [review]

**Review by CuTungBo88 — COMMENTED** [review]
Ok wow

**Comment by cooltey** [comment]
> @cooltey Just a couple of conflicts, otherwise lgtm

Done!

**Pull request merged**